tobac/plotting.py

```diff
@@ -1730,8 +1730,45 @@ def plot_mask_cell_track_static_timeseries(
         plt.clf()
 
 
-def map_tracks(track, axis_extent=None, figsize=(10, 10), axes=None):
+def map_tracks(
+    track, axis_extent=None, figsize=None, axes=None, untracked_cell_value=-1
+):
+    """Maps the tracks of all tracked cells
+
+    Parameters
+    ----------
+    track: pandas dataframe
+        Tracks from tobac
+    axis_extent: array-like, length 4
+        Extent of the map, as required by `cartopy`
+    figsize: depreciated
+        Depreciated parameter
+    axes: Matplotlib axes or geoaxes from cartopy
+        Axes to plot the tracks onto
+    untracked_cell_value: int or np.nan
+        Untracked cell value from tobac.
+
+    Returns
+    -------
+    Returns `axes` with the tracks plotted onto it.
+
+    Raises
+    ------
+    Raises a `ValueError` if `axes` is not passed in.
+
+    """
+    if figsize is not None:
+        warnings.warn(
+            "figsize is depreciated as this function does not create its own figure.",
+            DeprecationWarning,
+        )
+    if axes is None:
```

Comment:
Is there a reason why we set `None` as a  default value for the `axes` parameter and then raise an error when no axis object is provided? I do not see the meaning right now, so maybe it is a leftover for when the function created its own figure?

Reply:
The ultimately correct way to do this would be to have `axes` as a required positional argument, but I believe this strange structure is leftover from when the function created its own figure. I wanted to add these in there as, without these errors, it will fail in a strange way. 

Given our headaches with `field_in` and the overall uncertain future of the plotting module, I am hesitant to move `axes` to being a required positional argument for now. I think it's something we should look into in the future as we figure out what we want to do with the plotting module long term.

Reply:
OK yes, sounds good! So from my side: ready to merge